include/lemlib/devices/gyro/gyro.hpp

```diff
@@ -42,23 +44,23 @@ class Gyro {
          *
          * @return float heading, in radians, locked from -pi to +pi
```

Comment:
```suggestion
         * @return Angle heading
```
Update doxygen comment